CHANGELOG_PENDING.md

```diff
@@ -19,6 +19,8 @@ program](https://hackerone.com/tendermint).
 
 ### IMPROVEMENTS:
 
+- [tests] [#\4322](https://github.com/tendermint/tendermint/pull/4322) Bind test servers to 127.0.0.1 (@erikgrinaker)
```

Comment:
no need to add a changelog entry for every change. if it becomes too big, no one will read it.